app/views/invitations/_invitation.html.erb

```diff
@@ -10,8 +10,9 @@
     <dd><%= invitation.invitee_email %></dd>
     <% else %>
     <dd>
+      <%= error_messages_for @invitation %>
       <%= form_for(invitation) do |f| %>
-        <p><label for="invitee_email">Enter an email address:</label> <%= f.text_field :invitee_email %></p>
+        <p><%= label_tag "invitation[invitee_email]", t('.email_address_label') %>: <%= f.text_field :invitee_email %></p>
```

Comment:
`f.label :invitee_email` should work here, and it would be more consistent since we're using `f.text_field :invitee_email`. If you could also use double quotes around the translation key name and remove the colon (we've been slowly eliminating them on our form labels, plus taking it out of the actual label means it doesn't get the red highlight when there's an error), that would be great!

Reply:
Just to make sure, do you want the "Enter an email address" label to be in red as well, like this? ![2](https://github.com/otwcode/otwarchive/assets/40988246/271d35ca-4761-4c76-85ac-5d6fe3adda8a)

Reply:
Yup, that's fine!